03-Bash/find_dup_files/find_dup_files.sh

```diff
@@ -7,7 +7,7 @@ function view_help {
 	echo "Script find_dup_files.sh usage information."
 	echo "SYNOPSIS"
 	echo "find_dup_files.sh [DIR] [-h]"
-	echo -e "Display information about files with identical md5, if the directory is not specified, then the current directory is by default.\n"
+	echo -e "Display information about files with identical md5,\n if the directory is not specified, then the current\n directory is by default.\n"
```

Comment:
Please check your commit using checkpatch.pl and fix warnings.

Reply:
Ok. Thank you. I fixed it used /scripts/checkpatch.pl -g HEAD, linux version 4.14 and updated the branch